### PR TITLE
Bottom nav

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check

`src/components/ProtectedRoute.jsx`
`src/components/PublicRoute.js`
`src/components/bottomNavigation/BottomNavigation.jsx`
`src/context/like-context.js`

### [Live](https://deploy-preview-11--play-arts-updated.netlify.app/)

Thank you for reviewing PR :)

### CODE REVIEW

File: src/components/bottomNavigation/BottomNavigation.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export const BottomNavigation = () =>{
+    return (<nav className="bottom-nav">
+        <ul className={styles.bottomNavWrapper}>
+        {
+            asideData.map(item => {
+                return (
+                    <li key={item.id}><NavLink to={item.linkTo} className="light-text" exact>{item.icon}</NavLink></li>
+                )
+            })
+        }
```
Comment (BalajiHarinath) [inline src/components/bottomNavigation/BottomNavigation.jsx:13]: Appreciation: Used map, used NavLink instead of Link

**Review by BalajiHarinath — APPROVED** [review]
Everything looks good, nothing to suggest.

File: src/context/like-context.js [hunk 2]
```diff
@@ -44,13 +44,22 @@ const LikeProvider = ({ children }) => {
...
         isOpen: "true",
         bg: "Red",
       }));
+    }
+      else {setToastVal((prevVal) => ({
+        ...prevVal,
+        msg: "Already in liked videos",
+        isOpen: "true",
+        bg: "Red",
+      }));}
```
Comment (anik31) [inline src/context/like-context.js:62]: appreciation - nice that you're doing proper error handling

File: src/components/cards/SingleCard.jsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ export const SingleCard = ({ item, flag }) => {
 
   return (
     <div className={styles.videoCard} key={item._id}>
-      <NavLink to={`/${item._id}`} className="decor-none light-text">
+      <NavLink to={`/video/${item._id}`} className="decor-none light-text">
         <img src={item.gif} className={styles.image} />
```
Comment (anik31) [inline src/components/cards/SingleCard.jsx]: suggestion - add alt text to img tags

**Review by anik31 — APPROVED** [review]
Nice work. Code is very clean & UI is also good.

File: src/components/cards/SingleCard.jsx (hunk 3 above)
Comment (Developers-codz) [reply src/components/cards/SingleCard.jsx]: Yep will do that, Thanks for pointing it out

**Pull request merged**